maintenance_page/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM --platform=${BUILDPLATFORM} bitnami/nginx:1.27.0-debian-12-r6
```

Comment:
Had to add the platform option otherwise it builds locally for an M1 Mac which then won't deploy to Cloud Platform